ui/raidboss/data/04-sb/dungeon/kugane_castle.js

```diff
@@ -56,18 +62,217 @@ export default {
+        'The Noh Theater': 'Théâtre de nô',
+        'Yojimbo': 'Yojimbo',
+        'Zuiko-Maru': 'Zuiko Maru',
+      },
+      'replaceText': {
+        'Clearout': 'Fauchage',
+        'Clockwork Medium': 'Kuchiyose mécanique',
+        'Clockwork Raiton': 'Raiton mécanique',
+        'Dragon\'s Lair': 'Fléau du dragon',
+        'Dragonfire': 'Feu du dragon',
+        'Dragonstrike': 'Frappe du dragon',
+        'Fragility': 'Fragilité',
+        'Gratuity': 'Gracieuseté',
+        'Harakiri': 'Hara-kiri',
+        'Helm Crack': 'Fendeur de casque',
+        'Iai-Giri': 'Iai-giri',
+        'Inoshikacho': 'Ino-shika-cho',
+        'Issen': 'Issen',
+        'Juji Shuriken': 'Juji Shuriken',
+        'Kenki Release': 'Décharge Kenki',
+        'Metta-Giri': 'Metta-giri',
+        'Tatami-Gaeshi': 'Tatami-gaeshi',
+        'Wakizashi': 'Wakizashi',
+        'Zanmato': 'Zanmato',
+        'Zeni Masshigura': 'Zeni Masshigura',
+      },
+    },
+    {
+      'locale': 'ja',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Daigoro': 'ダイゴロウ',
+        'Dojun-Maru': 'ドウジュン丸',
+        'Elite Onmitsu': 'ジョウ・オンミツ',
+        'Kageyama': 'カゲヤマ',
+        'The Budokan Training Grounds': '天守武道館',
+        'The Keisen Garden': '渓泉園',
+        'The Noh Theater': '天守能楽堂',
+        'Yojimbo': 'ヨウジンボウ',
+        'Zuiko-Maru': 'ズイコウ丸',
+      },
+      'replaceText': {
+        'Clearout': 'なぎ払い',
+        'Clockwork Medium': 'からくり口寄せ',
+        'Clockwork Raiton': 'からくり雷遁',
+        'Dragon\'s Lair': '雲蒸竜変',
+        'Dragonfire': '暴竜',
+        'Dragonstrike': '画竜点睛',
+        'Fragility': '儚き命',
+        'Gratuity': '心づけ',
+        'Harakiri': '腹切り',
+        'Helm Crack': '兜割り',
+        'Iai-Giri': '居合斬り',
+        'Inoshikacho': '猪鹿蝶',
+        'Issen': '一閃',
+        'Juji Shuriken': '十字手裏剣',
+        'Kenki Release': '剣気解放',
+        'Metta-Giri': '居合滅多斬り',
+        'Tatami-Gaeshi': '畳返しの術',
+        'Wakizashi': '脇差',
+        'Zanmato': '斬魔刀',
+        'Zeni Masshigura': '銭まっしぐら',
+      },
+    },
+    {
+      'locale': 'cn',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Daigoro': '大五郎',
+        'Dojun-Maru': '道顺丸',
+        'Elite Onmitsu': '攘夷隐密',
+        'Kageyama': '景山',
+        'The Budokan Training Grounds': '天守武道馆',
+        'The Keisen Garden': '溪泉园',
+        'The Noh Theater': '天守能乐堂',
+        'Yojimbo': '保镖',
+        'Zuiko-Maru': '瑞光丸',
+      },
+      'replaceText': {
+        'Clearout': '横扫',
+        'Clockwork Medium': '机关召集',
+        'Clockwork Raiton': '机关雷遁',
+        'Dragon\'s Lair': '云蒸龙变',
+        'Dragonfire': '龙火',
+        'Dragonstrike': '画龙点睛',
+        'Fragility': '人生短暂',
+        'Gratuity': '赏钱',
+        'Harakiri': '切腹',
+        'Helm Crack': '破盔',
+        'Iai-Giri': '居合斩',
+        'Inoshikacho': '猪鹿蝶',
+        'Issen': '一闪',
+        'Juji Shuriken': '十字手里剑',
+        'Kenki Release': '剑气解放',
+        'Metta-Giri': '居合多段斩',
+        'Tatami-Gaeshi': '',
```

Comment:
Why empty?
also `ko`

Reply:
ahh .... forgott to fix it im my tools xD